newIDE/app/src/UI/Theme/i18n.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import domain from './domain';
+
+const directionSpec = {
+  get direction() {
+    return domain['language-ltr'] ? 'unset' : 'rtl';
+  },
+};
+
+export const text = {
+  root: { root: directionSpec },
+  label: { label: directionSpec },
+};
+
+const orderSpec = {
+  get order() {
+    return domain['language-ltr'] ? 'unset' : (styles.order?.rtl ?? 100);
+  },
+};
+
+export const icon = {
+  root: { root: orderSpec },
+};
+
+const styles = {
+  order: {
+    rtl: 100,
+  },
+};
+
+export default styles;
+
+export const formula = {
```

Comment:
Avoid calling things with generic name like "formula", especially if you rename them later when importing them.
Note that this export and object might not be needed if you switch to a `convertToRtl` approach.